Sources/NIO/CircularBuffer.swift

```diff
@@ -50,36 +50,63 @@ public struct CircularBuffer<E>: CustomStringConvertible, AppendableCollection {
     ///
     /// Amortized *O(1)*
     public mutating func append(_ value: E) {
+        return self.addLast(value)
+    }
+
+    /// Add an element to the end of the ring buffer.
+    ///
+    /// Amortized *O(1)*
+    public mutating func addLast(_ value: E) {
```

Comment:
why do we have this method? We have `append` already which matches `Array`'s `append`.

Reply:
@weissi I added it for consistent naming ;)